src/amo/components/AddonVersionCard/index.js

```diff
@@ -27,17 +27,27 @@ type Props = {|
   // An undefined version means the versions are still loading, whereas a null
   // version means that no version exists.
   version: AddonVersionType | null | void,
+  showButtonInsteadOfLink: boolean,
```

Comment:
I would call it `showLinkInsteadOfButton` because in general one would expect a button and the link would be the exception. For the same reason, I would default its value to `false`.